specification/common-types/resource-management/v3/types.json

```diff
@@ -282,6 +382,7 @@
         },
         "percentComplete": {
           "type": "number",
+          "format": "double",
```

Comment:
I'm not sure if it matters. Need to confirm.

Reply:
Well confirmed it matters.

Reply:
does this mean that this line shouldn't be there? 

we already had a conversation about this particular line: https://github.com/Azure/typespec-azure/issues/832

and I am following this up in: https://github.com/Azure/typespec-azure/issues/2041

Reply:
I'm a little bit curious in the details of how exactly it matters, and whether we're saying it should be a double or not... In underlying json format there's no real distinction between number types. So, shouldn't all numbers be assumed to be doubles unless there's good reason otherwise? And percentages seem like a very general pattern of number that already represents a fraction that could want to include double values for precision. But is changing it to double going to break a lot of SDKs? Do those SDKs just automatically round double values today if they get them?

Reply:
side note: I talked with Pan about this offline and she is ok with keep `"format": "double",`